test-requirements.txt

```diff
@@ -5,5 +5,5 @@ coverage
 fixtures~=4.0.0
 pylint
 pytest
-requests-mock
+requests-mock >= 1.11.0
 tox
```

Comment:
[Renovate doesn't know about this file](https://docs.renovatebot.com/python/).  Perhaps I should teach it to process this file the same as it does for `requirements.txt`?

That way we know about dep. updates that impact CI.